csfieldguide/static/interactives/pixel-viewer/js/pixel-viewer.js

```diff
@@ -151,7 +151,7 @@ function setUpMode(){
 
```

Comment:
![image](https://user-images.githubusercontent.com/25916475/58214707-e2584b80-7d4a-11e9-9558-df04627a1d38.png)
Can these in threshold mode be positioned above eachother?

Reply:
This also looks terrible, will update.